addons/hw_drivers/controllers/driver.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python3
 import logging
 import time
+import inspect
+from abc import ABCMeta
```

Comment:
What is the purpose of this? pretty sure it's not needed

Reply:
This is part of the serial drivers PR. since those commits are not merged in master, they appear here because I need them, but they should not be part of this PR.

As for the use of ABCMeta, it makes it possible to check if a class is abstract, via the presence of an @abstractmethod decorator on abstract methods. Base classes are kept out of the drivers register.

Reply:
@pha-odoo , can you mention me this PR, it could be great to change the way it works, as your real abstract class which is 'Driver', doesn't use any of this mechanism, I think you should not use this abstract stuff in SerialDriver in particular

Reply:
Serial dirvers PR: #33800